flink-tests/src/test/java/org/apache/flink/test/recovery/BatchFineGrainedRecoveryITCase.java

```diff
@@ -199,6 +371,76 @@ private boolean toss() {
 		}
 	}
 
+	private enum StaticMapFailureTracker {
+		;
+
+		private static final List<AtomicInteger> mapRestarts = new ArrayList<>(MAP_NUMBER);
+		private static final List<AtomicInteger> expectedMapRestarts = new ArrayList<>(MAP_NUMBER);
+		private static final List<AtomicInteger> mapFailures = new ArrayList<>(MAP_NUMBER);
```

Comment:
mapFailures and mapFailuresWithBacktracking  are not used in any assertion

Reply:
I wanted to keep them for logs and debuging